frontend/src/components/SortableTask.module.scss

```diff
@@ -34,3 +34,23 @@
     height: 100%;
   }
 }
+
+.taskDetailsTooltip {
+  @extend .layout-column;
+  align-items: center;
+  gap: 20px;
+
+  .title {
+    margin-bottom: 2px;
+  }
+
+  .customerValues {
+    @extend .layout-row;
+    gap: 15px;
+
+    & > div {
+      @extend .layout-row;
+      gap: 4.75px;
```

Comment:
I'm not sure if it's just me, but splitting pixels feels a bit odd. Could we manage with a whole number?

Reply:
The value is probably from Figma, but could be rounded.